### PR TITLE
Permission edit all workshops

### LINKED ISSUE #27: Admin should be able to view and edit workshops

### PR DESCRIPTION
**Pull request by cytadela8**
Simple change to allow admins to edit all workshops.

This closes #27

---
This change is [Reviewable](https://reviewable.io/reviews/warsztatywww/aplikacjawww/66)

### CODE REVIEW

File: wwwapp/views.py [hunk 1]
```diff
@@ -165,28 +165,25 @@ def my_profile_view(request):
     return render(request, 'profile.html', context)
 
 
+@login_required()
```
Comment (grzegorz225) [inline wwwapp/views.py]: Do we want to enforce logins on this view? It wasn't enforced before.

File: wwwapp/views.py [hunk 2]
```diff
@@ -256,7 +253,8 @@ def workshop_page_view(request, name):
 
 def can_edit_workshop(workshop, user):
     if user.is_authenticated:
-        return Workshop.objects.filter(id=workshop.id, lecturer__user=user).exists()
+        return Workshop.objects.filter(id=workshop.id, lecturer__user=user).exists() \
+               or request.user.has_perm('wwwapp.edit_all_workshops')
```
Comment (krzys-h) [inline wwwapp/views.py]: request doesn't even exist here XD

```
NameError at /workshop/test/
name 'request' is not defined
```

**Comment by cytadela8** [comment]
After #69 this requires significant changes to handle correctly admins with and without permission to edit workshops.

**Comment by cytadela8** [comment]
I think I don't really have time to work on this now.

**Comment by krzys-h** [comment]
The editors are now separated so we can finally do this. I rebased the PR on latest master because there was a bunch of merge conflicts.

**Pull request merged**